homeassistant/components/fritz/common.py

```diff
@@ -107,20 +109,25 @@ def unique_id(self):
         return self._unique_id
 
     @property
-    def fritzbox_model(self):
+    def model(self):
         """Return model."""
-        return self._device_info["model"].replace("FRITZ!Box ", "")
+        return self._model
 
     @property
-    def device_info(self):
-        """Return device info."""
-        return self._device_info
+    def sw_version(self):
```

Comment:
Drop the property and just set `self.sw_version` as nothing is happening in the property

Reply:
Done